src/interfaces/csharp/swig_typemaps.i

```diff
@@ -190,18 +190,17 @@ TYPEMAP_SGMATRIX(float64_t, double, double)
 		return $null;
 	}
 
-	shogun::SGString<SGTYPE>* strings=SG_MALLOC(shogun::SGString<SGTYPE>, rows_$1);
+	shogun::SGVector<SGTYPE>* strings=SG_MALLOC(shogun::SGVector<SGTYPE>, rows_$1);
```

Comment:
these things can be tested with appropriate meta examples / integration tests... and should be!